keras_hub/src/models/pali_gemma/pali_gemma_presets.py

```diff
@@ -219,3 +222,34 @@
         "kaggle_handle": "kaggle://keras/paligemma2/keras/pali_gemma2_pt_28b_896/1",
     },
 }
+
+# Ensure compatibility with the official naming convention.
```

Comment:
I think we should keep this a lot simpler, just a list of aliases and resolved names. No regexes.

We should also figure out how we don't list the old name in `class.presets`, that will prevent them from being rendered on keras.io. We don't want to document the old names presumably.

Reply:
Maybe we could add something to `pali_gemma/__init__.py`.

```python
from keras_hub.src.utils.preset_utils import register_presets
from keras_hub.src.utils.preset_utils import register_aliases

# Old names for presets we no longer use but still support.
register_aliases([
    ("pali_gemma2_10b_ft_docci_448", "pali_gemma2_ft_docci_10b_448"),
])
```